addons/io_hubs_addon/components/ui.py

```diff
@@ -142,6 +145,271 @@ def draw(self, context):
         draw_components_list(self, context)
 
 
+def export_scene():
+    try:
+        import os
+        import sys
+        extension = '.glb'
+        output_dir = bpy.app.tempdir
+        if not os.path.exists(output_dir):
+            os.makedirs(output_dir)
+        args = {
+            # Settings from "Remember Export Settings"
+            **dict(bpy.context.scene.get('glTF2ExportSettings', {})),
+
+            'export_format': ('GLB' if extension == '.glb' else 'GLTF_SEPARATE'),
+            'filepath': os.path.join(bpy.app.tempdir, EXPORT_TMP_FILE_NAME),
+            'export_cameras': True,
+            'export_lights': True,
+            'export_extras': True,
+            'use_visible': True,
+            'export_apply': True
+        }
+        bpy.ops.export_scene.gltf(**args)
+    except Exception as err:
+        print(err, file=sys.stderr)
```

Comment:
I don't think we should just print the error and continue on from this, if the glb export fails then the update process should be stopped and the user notified.

Reply:
Thanks!  Yeah, the `hubs_report_viewer` does a good job :)